dart/lib/src/isolate_error_integration.dart

```diff
@@ -10,21 +10,23 @@ import 'sentry_options.dart';
 import 'throwable_mechanism.dart';
 
 class IsolateErrorIntegration extends Integration {
-  late RawReceivePort _receivePort;
+  RawReceivePort? _receivePort;
 
   @override
   FutureOr<void> call(Hub hub, SentryOptions options) async {
     _receivePort = _createPort(hub, options);
 
-    Isolate.current.addErrorListener(_receivePort.sendPort);
+    Isolate.current.addErrorListener(_receivePort!.sendPort);
```

Comment:
This should be guarded by an `if`.

Reply:
The `_createPort` method returns an object of type `RawReceivePort` and not `RawReceivePort?`, so, I don't see why it's necessary.

Reply:
In that case the method signature of `_createPort ` should be updated

Reply:
Is there any way to change this pull request instead of creating another one? Sorry, I'm not used to making contributions that much.

Reply:
No, worries. You can push more changes to your branch. The new changes then show up here.